### ISSUE TITLE
Qcma crashes on startup

### ISSUE THREAD

**Issue by OvanH** [codestation/qcma#50]
Qcma startup 1 time, i got through the first start configuration and the it would not start any more, i have tried deleting al the preferences and application support files and downloading a new version but to no avail.

crash report gives main reason as: 

Dyld Error Message:
  Symbol not found: _OSAtomicIncrement32Barrier
  Referenced from: /Users/USER/Desktop/Qcma.app/Contents/MacOS/../Frameworks/libusb-1.0.0.dylib
  Expected in: /usr/lib/libSystem.B.dylib

i am on osx 10.8.5.

**Comment by codestation** [comment]
Have you tried with a previous version? you can find them in http://codestation.nekmo.com/qcma/ under the osx folder. I compiled the latest one with newer OSX version since i don't have access to a 10.8 environment anymore, so probably the newer toolchain dropped support for it.

Apple also dropped support for OSX 10.8 on mid 2015 so cannot guarantee that newer Qcma versions will work on that OS version.

**Comment by OvanH** [comment]
Thank you for the link, i could not find older versions. Will try this afternoon.

**Comment by OvanH** [comment]
0.3.9 at least starts and seams to work. my memory card died last night so i have to wait on my new one to truly test.

thank you

**Comment by OvanH** [comment]
it sees the vita and my pan name but no memory card cuz its dead. But I can go through all the settings and qcma doesn't crash.

**Comment by metalminx** [comment]
i have the same prob. should i open a new issue or is using this thread OK?
my issue is i am on MacOS 10.7.5 and QCMA 0.3.9 worked perfectly until i had to uninstall it when my vita battery burned out. i just got a new one, and 0.3.9 will not boot anymore. crashes on start up. i've tried other versions above and below and same problem.
CRASH REPORT: 
Process:         launchd [9641]
Path:            /Applications/Qcma.app/Contents/MacOS/qcma
Identifier:      com.yourcompany.qcma
Version:         ??? (???)
Code Type:       X86-64 (Native)
Parent Process:  launchd [138]

Date/Time:       2017-01-07 21:34:25.650 -0500
OS Version:      Mac OS X 10.7.5 (11G63b)
Report Version:  9

Interval Since Last Report:          64853167 sec
Crashes Since Last Report:           250
Per-App Crashes Since Last Report:   53
Anonymous UUID:                      82768200-62B9-4EB3-905B-27E1726102BD

Crashed Thread:  Unknown

Exception Type:  EXC_BAD_ACCESS (SIGSEGV)
Exception Codes: KERN_INVALID_ADDRESS at 0x00007fff5fc01028

Backtrace not available

Unknown thread crashed with X86 Thread State (64-bit):
  rax: 0x0000000000000055  rbx: 0x0000000000000000  rcx: 0x0000000000000000  rdx: 0x0000000000000000
  rdi: 0x0000000000000000  rsi: 0x0000000000000000  rbp: 0x0000000000000000  rsp: 0x0000000000000000
   r8: 0x0000000000000000   r9: 0x0000000000000000  r10: 0x0000000000000000  r11: 0x0000000000000000
  r12: 0x0000000000000000  r13: 0x0000000000000000  r14: 0x0000000000000000  r15: 0x0000000000000000
  rip: 0x00007fff5fc01028  rfl: 0x0000000000010203  cr2: 0x00007fff5fc01028
Logical CPU: 0

Binary images description not available


External Modification Summary:
  Calls made by other processes targeting this process:
    task_for_pid: 1
    thread_create: 0
    thread_set_state: 0
  Calls made by this process:
    task_for_pid: 0
    thread_create: 0
    thread_set_state: 0
  Calls made by all processes on this machine:
    task_for_pid: 62388
    thread_create: 0
    thread_set_state: 0

Model: Macmini5,1, BootROM MM51.0077.B10, 2 processors, Intel Core i5, 2.3 GHz, 4 GB, SMC 1.76f0
Graphics: Intel HD Graphics 3000, Intel HD Graphics 3000, Built-In, 384 MB
Memory Module: BANK 0/DIMM0, 2 GB, DDR3, 1067 MHz, 0x802C, 0x31364A5346323536363448592D3147314431
Memory Module: BANK 1/DIMM0, 2 GB, DDR3, 1067 MHz, 0x802C, 0x31364A5346323536363448592D3147314431
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0xE4), Broadcom BCM43xx 1.0 (5.106.198.19.22)
Bluetooth: Version 4.0.8f17, 2 service, 11 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en1
Serial ATA Device: Hitachi HTS545050B9A302, 500.11 GB
USB Device: hub_device, 0x0424  (SMSC), 0x2513, 0xfd100000 / 2
USB Device: IR Receiver, apple_vendor_id, 0x8242, 0xfd110000 / 3
USB Device: hub_device, 0x0424  (SMSC), 0x2513, 0xfa100000 / 2
USB Device: Hub in Apple Extended USB Keyboard, apple_vendor_id, 0x1002, 0xfa120000 / 4
USB Device: Apple Optical USB Mouse, apple_vendor_id, 0x0307, 0xfa122000 / 6
USB Device: Apple Extended USB Keyboard, apple_vendor_id, 0x0204, 0xfa121000 / 5
USB Device: BRCM20702 Hub, 0x0a5c  (Broadcom Corp.), 0x4500, 0xfa110000 / 3
USB Device: Bluetooth USB Host Controller, apple_vendor_id, 0x8281, 0xfa113000 / 7
FireWire Device: OXFORD IDE Device, Oxford Semiconductor Ltd., 400mbit_speed

trying to use QCMA again for transferring large 3GB+ VPKs. (Filezilla doesn't seem to like large files for this purpose.
![screen shot 2017-01-07 at 9 35 29 pm](https://cloud.githubusercontent.com/assets/24856882/21746852/91daeefc-d521-11e6-954d-36abbb1f248f.png)
